packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -38,7 +38,7 @@ const Div100vhContainer = ({
         height: max_autoheight_offset ? '' : height_rule,
         maxHeight: max_autoheight_offset ? `calc(100rvh - ${max_autoheight_offset})` : '',
     };
-    if (is_bypassed) return children;
+    if (is_bypassed) return children as JSX.Element;
```

Comment:
```suggestion

    if (is_bypassed) return children;
    
```
@likhith-deriv Why force casting here? I don’t have any errors 🤔

Reply:
The component that uses `Div100vhContainer` gets a error stating that Div100vhContainer is not a JSX element. Hence I either typecast it or wrap children in Ract.Fragment.
@mariya said she faced issues when children were wrapped with fragment. Hence went with typecasting approach
<img width="880" alt="Screen Shot 2022-11-16 at 12 15 11 PM" src="https://user-images.githubusercontent.com/98398322/202125102-48efab27-4713-48cd-a2ed-8296b167bb7c.png">